lib/gui/app/modules/drive-scanner.js

```diff
@@ -23,7 +23,7 @@ const permissions = require('../../../shared/permissions')
 const scanner = SDK.createScanner({
   blockdevice: {
     get includeSystemDrives () {
-      return settings.get('unsafeMode')
+      return settings.get('unsafeMode') || process.env.ETCHER_HIDE_UNSAFE_MODE
```

Comment:
This would turn unsafeMode **on** when this env var is set to any value – iirc we want it to be off? @curcuz?